eprihlaska/views.py

```diff
@@ -1241,3 +1241,15 @@ def admin_impersonate_user(id):
     session.clear()
     login_user(user)
     return redirect(url_for('index'))
+
+
+@app.template_filter('get_user')
+def get_user_filter(user_id):
```

Comment:
This deserves its own file, e.g. template_filters.py. Maybe worth refactoring in the future.